throttle.js

```diff
@@ -41,29 +51,21 @@ export default function (delay, noTrailing, callback, debounceMode) {
 		cancelled = true;
 	}
 
-	// `noTrailing` defaults to falsy.
-	if (typeof noTrailing !== 'boolean') {
-		debounceMode = callback;
-		callback = noTrailing;
-		noTrailing = undefined;
-	}
-
 	/*
 	 * The `wrapper` function encapsulates all of the throttling / debouncing
 	 * functionality and when executed will limit the rate at which `callback`
 	 * is executed.
 	 */
 	function wrapper(...arguments_) {
```

Comment:
ESLint complains about complexity loudly.
I'll refactor it if you want.
But it will not be so good since I don't have elegant ideas.

Reply:
Does it show warning or error? If it’s error, we can ignore it for now.

Reply:
It shows warning.
Can we ignore error, or is it a typo?

Reply:
Let’s ignore it.